app/views/layouts/application.html.erb

```diff
@@ -15,6 +16,8 @@
       <%= link_to t(".en"), locale: "en" %> |
       <%= link_to t(".vi"), locale: "vi" %>
     </span>
+    <%= render 'layouts/header' %>
```

Comment:
render "header" là dc rồi

Reply:
chị ơi em thấy các bạn đều dùng vậy với dùng mỗi header em thấy báo lỗi
ActionView::MissingTemplate in StaticPages#home